model/Dataset/Vocabulary/ConfidentialityLevelType.md

```diff
@@ -0,0 +1,22 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# ConfidentialityLevelType
+
+## Summary
+
+Categories of confidentiality level
+
+## Description
+
+Describes the different confidentiality levels as given by the [Trafic Light Protocol](https://en.wikipedia.org/wiki/Traffic_Light_Protocol)
+
+## Metadata
+
+- name: ConfidentialityLevelType
+
+## Entries
+
+- Red: Data points in the dataset are highly confidential and only can be shared with named recipients.
```

Comment:
Similar to the comment above for intended use -- is this Confidentiality Level Type meant to capture e.g. license restrictions on sharing of the data? If so, wouldn't a `LicenseRef-` be the preferred way to capture that?

Reply:
This is a standard vocabulary from the industry, which is orthogonal to actual license on the data.   Please open issue if you disagree with me marking this resolved.